suite-native/atoms/src/Skeleton/BoxSkeleton.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Easing } from 'react-native-reanimated';
+
+import {
+    Canvas,
+    useComputedValue,
+    vec,
+    useTiming,
+    RoundedRect,
+    RadialGradient,
+} from '@shopify/react-native-skia';
+
+import { useNativeStyles } from '@trezor/styles';
+import { nativeBorders } from '@trezor/theme';
+
+type BoxSkeletonProps = {
+    height: number;
+    width: number;
+    borderRadius?: number;
+};
+
+export const BoxSkeleton = ({
+    height,
+    width,
+    borderRadius = nativeBorders.radii.small,
+}: BoxSkeletonProps) => {
+    const {
+        utils: { colors },
+    } = useNativeStyles();
+    const gradientValue = useTiming(
+        { from: 0, to: width, loop: true, yoyo: true },
+        { easing: Easing.inOut(Easing.linear) },
```

Comment:
Can you please make animation to go only one direction? I never seen skeleton loader going back and forth before. It should go only forward to make user feel that loading is moving forwards.

Reply:
https://github.com/thevsstech/rn-skeleton this is how it should feel

Reply:
Skeleton reworked to move only to one direction in 6eb1cf3bf